sycl/include/CL/sycl/detail/spirv.hpp

```diff
@@ -216,24 +216,25 @@ EnableIfGenericBroadcast<T> GroupBroadcast(T x, id<Dimensions> local_id) {
 
 // Single happens-before means semantics should always apply to all spaces
 // Although consume is unsupported, forwarding to acquire is valid
+template <typename T>
 static inline constexpr __spv::MemorySemanticsMask::Flag
-getMemorySemanticsMask(ONEAPI::memory_order Order) {
+getMemorySemanticsMask(T Order) {
```

Comment:
Why need to introduce template version?
Is it not enough to change types here?

Reply:
@s-kanaev we need both types to preserve backwards compatibility